.github/workflows/build.yml

```diff
@@ -44,7 +46,7 @@ jobs:
     if: github.ref == 'refs/heads/main' || contains(github.ref, 'refs/tags')
     runs-on: ubuntu-22.04
     permissions:
-      contents: read
+      contents: write
```

Comment:
Nit: we could add a comment to document why `write` access is needed.

Reply:
Yep, added a comment to both.